testing/test_gateway.py

```diff
@@ -90,6 +91,7 @@ def test_gateway_status_busy(self, gw):
         ch2.waitclose()
         for i in range(10):
             status = gw.remote_status()
```

Comment:
``gw.remote_status()`` is a synchronous operation -- no time.sleep() should be neccessary.

Reply:
but it was already repeated 10 times - so it is waiting on something.

Reply:
See also the test run from the first commit:
https://travis-ci.org/pytest-dev/execnet/jobs/427581580

Reply:
i believe with the introduction of the execmodel backends behaviour changed the expectations and a wait-for with some sleep time is needed to accommodate for the extra sleep time